apps/public-reference/pages/applications/contact/alternate-contact-name.tsx

```diff
@@ -16,12 +16,16 @@ import {
 import FormsLayout from "../../../layouts/forms"
 import { useForm } from "react-hook-form"
 import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
-import ApplicationConductor from "../../../lib/ApplicationConductor"
-import { useContext, useMemo } from "react"
+import FormBackLink from "../../../src/forms/applications/FormBackLink"
 
 export default () => {
   const { conductor, application, listing } = useContext(AppSubmissionContext)
-  const currentPageStep = 1
+  const currentPageSection = 1
+
+  conductor.stepTo("Alternate Contact Name")
+  useEffect(() => {
+    conductor.skipCurrentStepIfNeeded()
+  }, [conductor])
```

Comment:
Maybe combine this into a custom hook so that we don't have to add this manually to every form component that might be optionally skipped?

```typescript
function useFormConductor(stepName: string) {
  const submissionContextValues = useContext(AppSubmissionContext)
  
  conductor.stepTo(stepName)
  
  useEffect(() => {
    conductor.skipCurrentStepIfNeeded()
  }, [conductor])
  return submissionContextValues
}

// in the component
const { conductor, application, listing } = useFormConductor("Alternate Contact Name")
```

I know it'll be a no-op for most components (e.g. the non-optional ones), but I think it will be potentially error prone to have to remember to add this effect by hand all the time to get automatic skipping to work.